build.py

```diff
@@ -155,10 +172,13 @@ def compile_variable_and_save(
 
     print(f"[{familyName}] Merging VTT")
     vttLib.transfer.merge_from_file(varFont, VTT_DATA_FILE)
+    if vtt_compile:
```

Comment:
mind adding in a log for Compiling VTT?

```suggestion
    if vtt_compile:
        print(f"[{familyName}] Compiling VTT")
```

Reply:
Eventually I'll get up the spare time to work a logging context into the multiprocessing steps so we can track time taken for each one ;P